manufacturing/internals/filter_panel/README.md

```diff
@@ -60,13 +44,37 @@ BEFORE purchasing any parts.**
 [a8mcmc]: https://www.mcmaster.com/92095A193/
 [a9mcmc]: https://www.mcmaster.com/92095A189/
 
-## Assembly
+## Custom parts
+
+This is the design for the custom cut acrylic plate for the filter housings.
+
+
+
+| Acrylic plate |
+:------------------|
+|![.iam](images/filter_panel_acrylic.jpg)|
+
+| Download Type | Link   |
+|:--------------|:-------|
+| Source: Inventor part file | [.ipt](filter_panel_acrylic_plate.ipt) |
+| Drawing                    | [.idw](filter_panel_acrylic_plate.idw) [.pdf](filter_panel_acrylic_plate.pdf)|
+| Export for laser cutting   | [.dxf](filter_panel_acrylic_plate.dxf) |
+
+The plate may be produced using a laser cutter or with common woodworking tools.
+
+
+## Filter Panel Assembly
```

Comment:
This should probably just say "Assembly"

Reply:
Also, seems somewhat inconsistent with where the "read general mfg methods" line goes (compare with enclosure page). Do we want it  right after the "Assembly" heading, or after the drawings right before verbal instructions?

Reply:
I think maybe before the drawing. Sort of as a "stop, read this before you start" check